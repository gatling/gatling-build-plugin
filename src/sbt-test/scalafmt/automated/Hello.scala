object Hello extends App { // append new line after object declaration
  final implicit val a: Long = 0L
  val b: Int = 1
}
