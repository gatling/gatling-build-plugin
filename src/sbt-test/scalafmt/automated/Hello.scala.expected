object Hello extends App {
  implicit final val a: Long = 0L
  val b: Int = 1 // Remove useless curly braces

  def min(p1: Int, p2: Int): Int = if (p1 < p2) {
    p1
  } else {
    p2
  }

  def remain(p1: Int, p2: Int): Boolean = try {
    p1 == p1 / p2 * p2
  } catch {
    case _: ArithmeticException => false
  }

  def withFor(n: Int): Seq[Int] = for {
    i <- 1 to n
  } yield { n }
}
